src/core/PDFContext.ts

```diff
@@ -163,6 +163,16 @@ class PDFContext {
     throw new UnexpectedObjectTypeError(types, result);
   }
 
+  getObjectRef(pdfObject: PDFObject): PDFRef {
+    for (const [ref, object] of this.indirectObjects.entries()) {
+      if (object === pdfObject) {
+        return ref;
+      }
+    }
+
+    throw new Error('Could not find PDFRef for PDFObject');
```

Comment:
This method seems likely to be used in cases where the `PDFObject` isn't yet registered in the context.
```suggestion
    return undefined;
```